.github/workflows/prerelease.yml

```diff
@@ -85,8 +85,8 @@ jobs:
           flavor: |
             latest=false
           tags: |
-            type=semver,pattern={{version}}-py${{ matrix.python-version }}
-            type=semver,pattern={{version}},enable=${{ matrix.python-version == 3.7 }}
+            type=semver,pattern={{version}}+py${{ matrix.python-version }}
+            type=raw,value={{version}}-py${{ matrix.python-version }}r
```

Comment:
Having a hard time parsing this. So before, we would tag as?

- v2.0.0-alpha.1-py3.7
- v2.0.0-alpha.1     (python 3.7 version)
- v2.0.0-alpha.1-py3.8
- v2.0.0-alpha.1-py3.9
- v2.0.0-alpha.1-py3.10

and now we'll tag as:

- v2.0.0-alpha.2+py3.7
- v2.0.0-alpha.2-py3.7r
- (no default "v2.0.0-alpha.2" image)
- v2.0.0-alpha.2+py3.8
- v2.0.0-alpha.2-py3.8r
- v2.0.0-alpha.2+py3.9
- v2.0.0-alpha.2-py3.9r
- v2.0.0-alpha.2+py3.10
- v2.0.0-alpha.2-py3.10r

Is that right?

Reply:
> * v2.0.0-alpha.1-py3.7
> * v2.0.0-alpha.1     (python 3.7 version)
> * v2.0.0-alpha.1-py3.8
> * v2.0.0-alpha.1-py3.9
> * v2.0.0-alpha.1-py3.10

We would _attempt_ this but the Docker metadata task would reject the `v2.0.0-alpha.1-py3.X`s because they aren't valid SemVer.

Instead, they all resulted in `v2.0.0-alpha.1`, see: https://github.com/nautobot/nautobot/actions/runs/4057206803/jobs/6983617197#step:8:45

> and now we'll tag as:
> 
> * v2.0.0-alpha.2+py3.7
> * v2.0.0-alpha.2-py3.7r
> * (no default "v2.0.0-alpha.2" image)
> * v2.0.0-alpha.2+py3.8
> * v2.0.0-alpha.2-py3.8r
> * v2.0.0-alpha.2+py3.9
> * v2.0.0-alpha.2-py3.9r
> * v2.0.0-alpha.2+py3.10
> * v2.0.0-alpha.2-py3.10r
> 
> Is that right?

Yes. Because this really isn't easily testable without attempting a release I feel this current syntax gives us the best option for the pre-release workflow not to fail. If for some reason the Docker metadata task doesn't like the `+` annotations, a "raw" tag of `v2.0.0-alpha.2-py3.7r` at least ensures _something_ will be tagged.

Reply:
Thanks for the info. I'm not sure I see the benefit/purpose of removing the default (non- `py3.x`) tag but the rest of it makes sense to me.

Reply:
> Thanks for the info. I'm not sure I see the benefit/purpose of removing the default (non- `py3.x`) tag but the rest of it makes sense to me.

Reverted the removal.